service/src/main/java/greencity/service/EventServiceImpl.java

```diff
@@ -1120,6 +1120,33 @@ public Page<UserProfilePictureDto> getUsersDislikedEventPage(Long eventId, Pagea
         return eventRepo.getUsersDislikedEventProfilePicturesPage(eventId, pageable);
     }
 
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public List<EventDto> getAllEventsOrganizedByUser(Long userId) {
+        User user = userRepo.findById(userId)
+            .orElseThrow(() -> new NotFoundException(ErrorMessage.USER_NOT_FOUND_BY_ID + userId));
+
+        List<Event> organizedEvents = eventRepo.getAllByOrganizer(user);
+
+        return organizedEvents.stream()
+            .map(event -> buildEventDto(event, userId))
+            .toList();
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public List<EventDto> getAllEventsAttendedByUser(Long userId) {
```

Comment:
can we create a query to get all user events(organized and attended) by userId?
In this case, we do not need extra calls to DB, like `userRepo.findById(userId) `and `userRepo.findById(userId)`
Can we have a situation when a user is an organizer of some event but now attended of that event?
If not we can get all attended an  event by a user and then check if the user is an organizer of the event